### ISSUE TITLE
Spell slots for Horizon Walker

### ISSUE THREAD

**Issue by SoilnRock** [larrychristensen/orcpub#123]
The spells gained as Horizon Walker shouldn't count for the normal spell limit of ranger spells. At lvl 5 I should get 4 lvl 1 spells and 2 lvl 2 spells + Protection from Evil and Good + Misty Step.
Orcpub only gives me 4 lvl 1 + 2 lvl 2 :-(

**Comment by DivertedCircle** [comment]
You're mixing up spells and spell slots. At level 5 you learn 4 Ranger spells plus your Horizon Walker spells but have 4 level 1 slots and 2 level 2 slots. Orcpub is correctly letting you choose 4 Ranger spells to learn on top of Protection From Evil and Good + Misty Step.

**Comment by SoilnRock** [comment]
Hi, thx for your reply! You're absolutely right - I live and I learn. Have a nice day :-)

**Comment by DivertedCircle** [comment]
The feat builder currently does not have the flexibility required to make this feat work. There may be a way to set it up via Selections but I'm unsure.

**Comment by SoilnRock** [comment]
Ah, that's very unfortunate, especially that there seems to be no way to add the druid cantrip when taking the Woof Elf Magic feat :-(

**Comment by DivertedCircle** [comment]
Feats can allow some spells to be learned with limitations. If you're choosing a Druid cantrip that uses an attack roll you could modify the feat to allow that by selecting 'Choose a class, gain a cantrip requiring an attack roll from that class's spell list.' in the spellcasting section of the feat builder.

**Comment by SoilnRock** [comment]
Yes, nice idea! Unfortunately the "attack roll" aspect limits the selection drastically.
It would be nice if the feat builder had an option of just selecting (any) cantrips/spells you learn when you choose the feat. Shouldn't be too hard tom implement, should it?
